chain/stmgr/stmgr.go

```diff
@@ -39,6 +40,8 @@ import (
 const LookbackNoLimit = api.LookbackNoLimit
 const ReceiptAmtBitwidth = 3
 
+const execTraceCacheSize = 16
```

Comment:
We probably want to make this configurable because SPs may want to disable it while exchanges may want to crank it up? We also are going with intuition for this value, so having ability to change it without a new build would help.

Reply:
Will make a followup PR which makes this tuneable via env variable so we can get this landed